File: 0231-power-of-two/solution.java
class Solution {
    public boolean isPowerOfTwo(int n) {
        double x=n;
         double c= Math.log10(n)/Math.log10(2);
   int a=(int)c;
   int m=1;
   for(int i=1;i<=a;i++){
    m=m*2;
   }
   if(m==n){
    return true;
   }else{
    return false;
   }
    } 
}

File: 0231-power-of-two/solution.cpp
class Solution {
public:
    bool isPowerOfTwo(int a) {
      if(a==0)return false;
    if(a==1){
        return true;
    }
    if(a%2==0){
        return isPowerOfTwo(a/2);
    }
    return false;

    }
};
